src/nn/tfjs/TfjsModel.js

```diff
@@ -50,11 +51,8 @@ export default class TfjsModel {
         await tf.setBackend('wasm');
       };
     } else {
-      if(tf.getBackend() != "WebGL"){
+      if(tf.getBackend() != "webgl"){
         await tf.setBackend('webgl');
-        tf.webgl.forceHalfFloat();
```

Comment:
does tfjs 2.0.1 support half float precision?

Reply:
It seems that versions after 1.7 will automatically be set to half float precision, there's no need to manually set.

Reply:
Thanks for this info. Where did you find that info? Could you please share the link? And please put that link into the new code as a comment. Then, I am fine to merge it. Thanks!

Reply:
The  problem was fixed in this[ PR](https://github.com/tensorflow/tfjs/pull/2410).